### PR TITLE
completed navbar

### PR DESCRIPTION
**Pull request by ZacWhittaker**
## Proposed changes

This adds a navbar with a logo to the frontend of the application. The navbar has been designed from the below figma template: https://www.figma.com/file/u6f4yX0ss98dYsz0H6vWvQ/FIT3170?node-id=139%3A5

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] DevOps related

## Checklist

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments
None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,16 +24,18 @@
...
   },
   "dependencies": {
-    "css-loader": "^5.2.1",
-    "file-loader": "^6.2.0",
     "@babel/cli": "^7.13.14",
     "@babel/core": "^7.13.14",
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-react": "^7.13.13",
     "babel-loader": "^8.2.2",
+    "bootstrap": "^4.6.0",
```
Comment (idomo1) [inline package.json:32]: Bootstrap is a pretty big dependency to bring in for a simple nav bar

File: src/components/App.jsx [hunk 2]
```diff
@@ -3,10 +3,12 @@ import { hot } from "react-hot-loader";
 import { BrowserRouter as Router, Route, Switch} from "react-router-dom";
 import Service from './Service';
 import UploadPage from './UploadPage';
-
+import Navbar from './Navbar'
+import "./navbar.css"
 function App(){
     return(
         <Router>
+            <Navbar></Navbar>
```
Comment (idomo1) [inline src/components/App.jsx:11]: This can be just `<Navbar />`

File: package.json (hunk 1 above)
Comment (idomo1) [reply package.json:32]: Are you even using this dependency? It doesn't look like it from the code.

**Review by idomo1 — CHANGES_REQUESTED** [review]
Please merge the dev branch into this branch, and make the navbar functional for the sections which currently exist.

**Comment by idomo1** [comment]
I notice as well that the navbar isn't mobile responsive

File: package.json (hunk 1 above)
Comment (ZacWhittaker) [reply package.json:32]: oh lol forgot to remove that, was trying out some stuff

File: package.json [hunk 3]
```diff
@@ -24,16 +24,17 @@
...
     "@babel/core": "^7.13.14",
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-react": "^7.13.13",
     "babel-loader": "^8.2.2",
+    "css-loader": "^5.2.1",
     "express": "^4.17.1",
+    "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.3.1",
     "react": "^17.0.2",
+    "react-bootstrap": "^1.6.0",
```
Comment (idomo1) [inline package.json:37]: Also this one doesn't appear to be used

**Review by crystaltr — COMMENTED** [review]
Same comments as Damien: 
- Unresponsive and is not resizing on mobile
- Feature button should jump to section not go to new page

**Comment by idomo1** [comment]
Nice work making it mobile responsive 👍 
From testing this out:
The navigation bar items should correspond to the sections (Upload, Example, and About). Pressing on an item should scroll the view to that section.
Service page is scrapped, so no need to navigate to it.

The mobile responsive items are off centre.
image

**Pull request closed without merging**